content/en/docs/concepts/cluster-administration/priority-and-fairness.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Priority and Fairness in Handling Requests
+content_template: templates/concept
+weight: 20
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+The control plane of Kubernetes is itself a resource (broadly
+construed) to which access is managed.  In particular, each apiserver
+applies some prioritization and fairness in its handling of requests.
+This includes not only kube-apiservers but aggregated apiservers too.
+Prioritization and fairness applies only to requests that are not
+_long running_.  Long running requests are WATCHes and EXECUTEs.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Status
+
+This feature is named "RequestManagement" and is alpha in
+release 1.16.  As an alpha feature, it is disabled by default.  When
+enabled it replaces a request filter that enforces simple aggregate
+concurrency limits with this feature, which can be seen as a more
+general version of that earlier functionality.
+
+## Outline
+
+Each apiserver independently applies prioritization and fariness,
+based on API objects that configure this feature.  This involves two
+concepts: priority level and flow schema.
+
+### Priority level
+
+Each apiserver has an aggregate concurrency limit and divides it ---
+without examining the request load --- among several priority levels.
+The priority levels operate independently of each other.  That is, the
+apiserver classifies a given request to exactly one priority level
+and, for a given priority level, handles at most the corresponding
+number of requests at any given time.  One special priority level ---
+used for meta-requests and other top-priority traffic --- imposes no
```

Comment:
How about:
```markdown
There is a special priority level, termed “exempt”, that does not impose any limit on concurrency.
```

Reply:
The original has a definition of "exempt" and says it applies to both priority levels and requests.